### PR TITLE
DEVO-732: Fixing publishing to nexus and updating the version to snap…

### PR DESCRIPTION
**Pull request by SameeraPriyathamTadikonda**
…shot

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ subprojects {
   apply plugin: "signing"
 
   group = "com.marklogic"
-  version = "1.3.0"
+  version = "1.3-SNAPSHOT"
```
Comment (jamesagardner) [inline build.gradle:7]: Shouldn't this be 1.3.0-SNAPSHOT if the eventual build will be 1.3.0 at release time?

I'm not sure we've setup real conventions on this project to say for certain.

File: marklogic-junit5/build.gradle [hunk 2]
```diff
@@ -86,11 +86,21 @@ publishing {
...
-        password mavenCentralPassword
+      if (project.hasProperty("mavenUser")) {
+        name = "nexus"
+        credentials {
+          username mavenUser
+          password mavenPassword
+        }
+        url publishUrl
+        allowInsecureProtocol = true
+      }else {
```
Comment (jamesagardner) [inline marklogic-junit5/build.gradle:97]: Should have a space after the close brace before the else here.

File: marklogic-unit-test-client/build.gradle [hunk 3]
```diff
@@ -84,11 +84,21 @@ publishing {
...
-        password mavenCentralPassword
+      if (project.hasProperty("mavenUser")) {
+        name = "nexus"
+        credentials {
+          username mavenUser
+          password mavenPassword
+        }
+        url publishUrl
+        allowInsecureProtocol = true
+      }else {
```
Comment (jamesagardner) [inline marklogic-unit-test-client/build.gradle:95]: Should have a space after the close brace and the else here.

File: build.gradle (hunk 1 above)
Comment (rjrudin) [reply build.gradle:7]: 1.3-SNAPSHOT is the convention, so this is good.

File: marklogic-unit-test-client/build.gradle (hunk 3 above)
Comment (SameeraPriyathamTadikonda) [reply marklogic-unit-test-client/build.gradle]: Fixed

**Review by rjrudin — APPROVED** [review]

**Pull request merged**